src/processes/domain-process.js

```diff
@@ -1,4 +1,5 @@
 const moment = require('moment');
+const {v4: uuidv4} = require('uuid');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `uuidv4` has been moved to the top of the file, which is a good practice as it improves readability and maintainability. All required modules should be imported at the beginning of the file.
```diff
+ const {v4: uuidv4} = require('uuid');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->